hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/MYSQL/V1_12_8__change_length_of_created_last_modified_by___MYSQL.sql

```diff
@@ -0,0 +1,45 @@
+ALTER TABLE sp_action MODIFY created_by VARCHAR(64);
```

Comment:
I think we could alter multiple columns at once using one SQL query:
ALTER TABLE sp_action ALTER COLUMN
created_by SET DATA TYPE VARCHAR(64),
last_modified_by SET DATA TYPE VARCHAR(64);